extensions/wikia/Recirculation/js/helpers/LiftigniterHelper.js

```diff
@@ -34,6 +34,7 @@ define('ext.wikia.recirculation.helpers.liftigniter', [
 
 			// Callback renders and injects results into the placeholder.
 			w.$p('register', registerOptions);
+			w.$p("setRequestFields", ["rank", "thumbnail", "title", "url", "presented_by"]);
```

Comment:
Hm maybe we should include `author` here, since I think that's part of the defaults. Too bad this completely replaces the options instead of just adding to it.